petab/problem.py

```diff
@@ -347,66 +347,111 @@ def get_noise_distributions(self):
         return measurements.get_noise_distributions(
             measurement_df=self.measurement_df)
 
+    def _to_mask(v: List, free: bool = True, fixed: bool = True):
+        """Apply mask of only free or only fixed values."""
+        if not free and not fixed:
+            raise ValueError("Disabling both `free` and `fixed` would "
+                             "yield an empty list.")
+        if not free:
+            return [v[ix] for ix in self.x_fixed_indices]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'v' is unsubscriptable](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4968439)